django/cantusdb_project/main_app/views/chant.py

```diff
@@ -1318,9 +1318,9 @@ def get_context_data(self, **kwargs):
 
         # 4064 is the id for the sequence database
         if source.segment.id == 4064:
-            queryset = source.sequence_set.order_by("folio", "sequence_number")
+            queryset = source.sequence_set.order_by("folio", "s_sequence")
         else:
-            queryset = source.chant_set.order_by("folio", "sequence")
+            queryset = source.chant_set.order_by("folio", "c_sequence")
```

Comment:
`"sequence_number"` and `"sequence"` were incorrect, these substitutions correct the error